UITests/Helper/TransferMethod/AddTransferMethod.swift

```diff
@@ -226,7 +226,7 @@ class AddTransferMethod {
     }
 
     func selectAccountType(_ accountType: String) {
-        accountTypeLabel.tap()
+        accountTypeLabel.doubleTap()
```

Comment:
Do we still need it if this didn't work?

Reply:
True.. will remove.